config/webpack.dev.config.js

```diff
@@ -43,52 +33,28 @@ const config = {
     rules: [
       {
         test: /\.(png|jpg|jpeg|gif|svg)$/,
-        use: [{
-          loader: 'url-loader',
-          query: {
-            limit: 8192,
-            name: 'assets/images/[name]__[hash:base64:5].[ext]'
-          }
-        }]
+        use: [{ loader: 'url-loader', options: { limit: 500000 } }]
```

Comment:
Hey @imlucas, I copied webpack 4 config from other plugin and have noticed that some plugins have different rules for `url-loader`.

First option:
```
use: [{
  loader: 'url-loader',
  query: {
    limit: 8192,
    name: 'assets/fonts/[name]__[hash:base64:5].[ext]'
  }
}]
```

Second option:
```
use: [{ loader: 'url-loader', options: { limit: 500000 } }]
```

Some pros and cons? What is preferable to use?